packages/app/src/features/dialogs/convert-stables/components/form/ConvertStablesForm.tsx

```diff
@@ -0,0 +1,57 @@
+import { UseFormReturn } from 'react-hook-form'
+
+import { Form } from '@/ui/atoms/form/Form'
+import { AssetSelectorWithInput } from '@/ui/organisms/asset-selector-with-input/AssetSelectorWithInput'
+import type { ConvertStablesFormSchema } from '../../logic/form/schema'
+import { ConvertStablesFormFields } from '../../types'
+
+export interface ConvertStablesFormProps {
+  formFields: ConvertStablesFormFields
+  form: UseFormReturn<ConvertStablesFormSchema>
+}
+
+export function ConvertStablesForm({ formFields, form }: ConvertStablesFormProps) {
+  const {
+    selectedAsset1,
+    selectedAsset2,
+    changeAsset1,
+    changeAsset2,
+    asset1Options,
+    asset2Options,
```

Comment:
nit:

```suggestion
    selectedFrom,
    selectedTo,
    changeFrom,
    changeTo,
    fromOptions,
    toOptions,
```
but please consider, cause it reads much better ;)